PD-1258-update-replication-ui-ref-content-about-dataset-properties (#3006)
truenas/documentation: content/SCALE/SCALEUIReference/DataProtection/ReplicationScreensSCALE.md

| **Source** | (Required) Enter or use <span class="material-icons">arrow_right</span> to the left of the **/mnt** folder and at each dataset to expand the dataset tree to browse to the dataset location that has snapshots to replicate. Click on the dataset or directory name, folder icon, or checkbox to select the dataset or directory. To enter multiple datasets, enter a comma (,) after each path in the **Source** field and then select another dataset. Click the <span class="material-icons"><span class="material-icons">arrow_drop_down</span></span> at the <span class="material-icons">folder</span>**/mnt** to collapse the dataset tree. |
| **Recursive** | Select to replicate all child dataset snapshots. When selected, **Exclude Child Datasets** displays. |
| **Exclude Child Datasets** | Displays after selecting **Recursive**. Enter the specific child dataset snapshots from the replication. Separate each entry by pressing <kbd>Enter</kbd>.  |

> | **Include Dataset Properties** | Select to include dataset properties with the replicated snapshots. This enables users to see ZFS dataset properties, which can be futher reviewed at the [ZFS manpages](https://linux.die.net/man/8/zfs).|

Reviewer: How about this?
```suggestion
| **Include Dataset Properties** | Select to include ZFS dataset properties with the replicated snapshots. For more information on ZFS dataset properties see [ZFS manpages](https://linux.die.net/man/8/zfs). |
```
